isso/utils/html.py

```diff
@@ -76,7 +76,7 @@ class Unofficial(misaka.HtmlRenderer):
 
     def blockcode(self, text, lang):
         lang = ' class="{0}"'.format(html.escape(lang)) if lang else ''
-        return "<pre><code{1}>{0}</code></pre>\n".format(html.escape(text,False), lang)
+        return "<pre><code{1}>{0}</code></pre>\n".format(html.escape(text, False), lang)
```

Comment:
https://github.com/posativ/isso/pull/680

Reply:
Jeeze, of COURSE there's already a PR for someone else making a minor spacing fix. :) FWIW this was put in by autopep8 which I ran to make Travis happy again.

Reply:
Speaking of travis... another construction site :(
https://www.jeffgeerling.com/blog/2020/travis-cis-new-pricing-plan-threw-wrench-my-open-source-works

Reply:
Eh, circleci seems like an okay alternative https://app.circleci.com/pipelines/github/ix5/isso/52/workflows/7f941e2b-00f5-44a4-b9c1-cb6b6d2e16d1

Reply:
Yeah, that'd be up to @posativ to change, though, I think? My mention of Travis was more just that the pipeline is still running (somehow) and it barfed on the flake8 failure.